### PR TITLE
refactor: change fontFamily in networkModal | ab

### PR DESCRIPTION
**Pull request by aliburzhi**

### CODE REVIEW

**Comment by aliburzhi** [comment]
Hey Marcus, i noticed that any time you click on li or button we got request for Font Open Sans, i can't understand why, i guess you know better the reasons , cos it is your code 
Снимок экрана 2022-09-23 в 16 45 46

**Comment by aliburzhi** [comment]
Снимок экрана 2022-09-23 в 16 50 29

**Comment by aliburzhi** [comment]
refactored, tested  and ready for review :)

**Comment by aliburzhi** [comment]
+ i fixed  finally (i really really hope) my settings in webstorm after i commited last changes

File: src/components/modal/modal.tsx [hunk 1]
```diff
@@ -83,8 +82,7 @@ export const Modal = ({
 			showModal={showModal}
 			background={background}
 			data-testid="modal-container"
-			setShowModal={setShowModal}
-		>
+			setShowModal={setShowModal}>
```
Comment (marcuszierke) [inline src/components/modal/modal.tsx:85]: the closing of the component should be on the next line as before - can you check in your settings?

**Review by marcuszierke — APPROVED** [review]

**Pull request merged**